cernopendata/opendata.cern.ch
# search: support old search querystring format

closes #3030

🆕 I accidentally found a `search_wrapper` function that does what we want so there's no need to have changes on Invenio-Search-UI.

#### Steps to test:

- All links following the old format e.g. `/search?type=Dataset` should be translated into the new one (`/search?f=type:Dataset`), thus making the search properly work.
- All the links displayed on the home page follow this old format, so clicking on them should perform the proper search.
- Play with different query params, you can get inspiration on the test included in this PR (`test_old_search_qs`).
